x-pack/plugins/security_solution/server/lib/app_features/app_features.ts

```diff
@@ -73,27 +73,31 @@ export class AppFeatures {
     const enabledSecurityAppFeaturesConfigs = this.getEnabledAppFeaturesConfigs(
       getSecurityAppFeaturesConfig(this.experimentalFeatures)
     );
-    this.featuresSetup.registerKibanaFeature(
-      this.securityFeatureConfigMerger.mergeAppFeatureConfigs(
-        securityBaseKibanaFeature,
-        securityBaseKibanaSubFeatureIds,
-        enabledSecurityAppFeaturesConfigs
-      )
+    const completeAppFeatureConfig = this.securityFeatureConfigMerger.mergeAppFeatureConfigs(
+      securityBaseKibanaFeature,
+      securityBaseKibanaSubFeatureIds,
+      enabledSecurityAppFeaturesConfigs
     );
 
+    this.logger.info(JSON.stringify(completeAppFeatureConfig));
```

Comment:
NIT: This can be a pretty big object, perhaps it would make more sense to log the `appFeaturesKeys` in the `set` method, the kibana feature config registered can be inferred from them. 
Otherwise, if this is needed for debugging purposes, we should change it to `debug` level.

Reply:
I'll change it to `debug()`. I felt like if something as not configured correctly in kibana, that having the full config dumped during start up would be something good to have.